scripts/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[workspace]
+resolver = "2"
+members = [
+    "check_copyright_header",
+]
+
+[workspace.dependencies]
+async-graphql = { version = "5.0.7", default-features = false }
+async-trait = "=0.1.58"
+bcs = "0.1.3"
+futures = "0.3.24"
+futures-util = "0.3.26"
+hex = "0.4.3"
+linera-sdk = { version = "0.2.0", path = "../linera-sdk" }
+linera-views = { version = "0.2.0", path = "../linera-views", default-features = false }
+log = "0.4.17"
+serde = { version = "1.0.152", features = ["derive"] }
+serde_json = "1.0.93"
+thiserror = "1.0.38"
+tokio = { version = "1.25.0", features = ["macros", "rt-multi-thread"] }
+webassembly-test = "0.1.0"
+regex = "1.4"
+anyhow = "1.0.57"
```

Comment:
I believe these are the only two useful dependencies.

Reply:
True, these were copy pasted and I forgot to remove the fat